src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,184 @@
 package core.basesyntax;
 
+import core.basesyntax.model.User;
+import core.basesyntax.service.ImpossibleRegisterUserException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorldTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User user = new User();
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+    }
+
+    @Test
+    void loginOfUserAlreadyExists_NotOk() {
+        user.setId(12l);
+        user.setLogin("sanya67");
+        user.setAge(18);
+        user.setPassword("123456");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("this login isn`t available");
+        }
+        User user1 = new User();
+        user.setId(12l);
+        user.setLogin("sanya67");
+        user.setAge(18);
+        user.setPassword("123456");
+        Assertions.assertThrows(ImpossibleRegisterUserException.class,
+                ()-> {
+                    registrationService.register(user1);
+                });
```

Comment:
This test correctly uses `Assertions.assertThrows` to verify that an `ImpossibleRegisterUserException` is thrown when a user with an existing login is registered. This approach should be consistently used across all tests where exceptions are expected.